runtime/queries/scss/highlights.scm

```diff
@@ -0,0 +1,86 @@
+(comment) @comment
+
+"~" @operator
+">" @operator
+"+" @operator
+"-" @operator
+"*" @operator
+"/" @operator
+"=" @operator
+"^=" @operator
+"|=" @operator
+"~=" @operator
+"$=" @operator
+"*=" @operator
+
+"in" @operator
+"and" @operator
+"or" @operator
+"not" @operator
+"only" @operator
+
+(tag_name) @tag
+(universal_selector) @tag
+(attribute_selector (plain_value) @string)
+(nesting_selector) @variable.other.member
+(pseudo_element_selector) @attribute
+(pseudo_class_selector) @attribute
+
+(identifier) @variable
+(class_name) @variable
+(id_name) @variable
+(namespace_name) @variable
+(property_name) @variable.other.member
+(feature_name) @variable
+(variable) @variable
+(variable_name) @variable.other.member
+(variable_value) @variable.other.member
+(argument_name) @variable.parameter
+(selectors) @variable.other.member
+
+(attribute_name) @attribute
+
+(function_name) @function
+
+((property_name) @variable
+ (match? @variable "^--"))
+((plain_value) @variable
+ (match? @variable "^--"))
```

Comment:
These need to be moved up (around L22 would do) because the stanzas above that match `property_name` and `plain_value` will take precedence. (In neovim the precedence is the other way around but in helix and all other tree-sitter consumers, the first stanza that matches [top to bottom] wins)